src/owl/stats/owl_stats.mli

```diff
@@ -492,11 +492,10 @@ from ``n`` trials. The probabilty mass function is as follows.
 .. math::
   P(x) = \frac{n!}{{x_1}! \cdot\cdot\cdot {x_k}!} p_{1}^{x_1} \cdot\cdot\cdot p_{k}^{x_k}
 
-``p`` is the probabilty mass of ``k`` categories. If the elements in ``p`` do
-not sum to 1, the last element of the ``p`` array is not used and is replaced
-with the remaining probability left over from the earlier elements.
+``p`` is the probabilty mass of ``k`` categories. The elements in ``p`` should
+all be positive (result is undefined in that case), but they don't need to sum
```

Comment:
What do you mean with `result is undefined in that case`?

Reply:
Woops, sorry the formulation is very unclear. I meant: "result is undefined if not".

Well there's a tradeof here: either we check that all elements in `p` are positive and raise `Invalid_argument` if not; either we check nothing but don't give any guarantee on the result if the precondition is not verified (I have no idea what the function does in that case, although I'm confident it terminates). The thing is that checking `p` at each call can be very costly, and for large scale simulation code you wouldn't want to do that. Another possibility would be to introduce a private type of float array with only positive values, so that the check can be done once and for all.

Reply:
I added a commit to improve the description in the doc.